src/SmartComponents/DriftPage/DriftTable/DriftTable.js

```diff
@@ -256,6 +301,8 @@ export class DriftTable extends Component {
     renderSystemHeaders() {
         let row = [];
         let typeIcon = '';
+        let referenceId = this.referenceId;
+        let updateReferenceId = this.updateReferenceId;
```

Comment:
I sometimes find casting instance variables to local variables unnecessary. The additional use of `this.` in this function to reference those instance variables is easier to parse when reading the code.